app/views/invitations/index.html.erb

```diff
@@ -1,16 +1,43 @@
-<fieldset>
+<% breadcrumb :invitations, space %>
```

Comment:
Maybe set this in the controller, instead of the view? But we set it in the view elsewhere, so it makes sense to put it here for consistency. Hrmph. 🤔